tests/connect_redirect/connect_redirect_tests.cpp

```diff
@@ -0,0 +1,482 @@
+// Copyright (c) Microsoft Corporation
+// SPDX-License-Identifier: MIT
+
+// This module facilitates testing various connect redirect scenarios by sending traffic to both
+// local system and a remote system, both running TCP / UDP listeners.
+
+#define CATCH_CONFIG_RUNNER
+
+#include "catch_wrapper.hpp"
+#include "ebpf_udp.h"
+#include "socket_helper.h"
+
+#include <chrono>
+#include <future>
+using namespace std::chrono_literals;
```

Comment:
https://github.com/microsoft/ebpf-for-windows/blob/main/docs/DevelopmentGuide.md#header-files says:

> DO include system headers (with <>) before local headers (with ""), and list them in alphabetical order where possible. This helps ensure there are not duplicate includes, and also helps ensure that headers are usable directly.